go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -2120,6 +2123,261 @@ func (s *VtctldServer) UpdateCellsAlias(ctx context.Context, req *vtctldatapb.Up
 	}, nil
 }
 
+// ValidateSchemaKeyspace is part of the vtctlservicepb.VtctldServer interface
+func (s *VtctldServer) ValidateSchemaKeyspace(ctx context.Context, req *vtctldatapb.ValidateSchemaKeyspaceRequest) (*vtctldatapb.ValidateSchemaKeyspaceResponse, error) {
+	shards, err := s.ts.GetShardNames(ctx, req.Keyspace)
```

Comment:
you still need to set up the tracing spans. you can look at any other function here for an example